AirCasting/APICommunicator/APIcalls.swift

```diff
@@ -0,0 +1,33 @@
+// Created by Lunar on 01/07/2021.
+//
+
+import Foundation
+
+final class APIcalls {
+    func forgotPassword(login: String, completion: @escaping (Int) -> Void) {
```

Comment:
I'm calling @mateuszmackowiak. I tried to add tips on how to refactor this func to be similar to one of yours but it surpassed me so - it's all yours 😎

Reply:
Mateusz's not here so I'll do my best to suggest a refactor ;D

Reply:
Well, without Mateusz I see it this way right now: 
It would be great to refactor this func to be consistent with the rest of API communication. It's not super important, and it's quite difficult. We could try pair programming or you can ask for Pawełs help :)  
But I think it's important to create something like Mateusz's "AuthorizationHTTPResponseHandler". I guess there could be responses like "wrong email" or "password can't be the same as previous one" and we should inform the user why it's not working. 
Am I right @pawelgil ?